src/components/datatable/DataTable.d.ts

```diff
@@ -82,6 +82,7 @@ declare class DataTable {
     $emit(eventName: 'filter', event: Event): this;
     $emit(eventName: 'value-change', value: any[]): this;
     $emit(eventName: 'row-click', event: Event): this;
+    $emit(eventName: 'row-mouseover', event: Event): this;
```

Comment:
I'm not saying this should be changed, but wouldn't it be better if the `event` parameter had the `MouseEvent` type since that's more accurate?

Reply:
then I should change the type to `MouseEvent` for more accurate right ?

Reply:
🤷‍♂️ Ultimately it's up to the repo people. I feel like it would be more accurate but they may prefer that you follow the pattern already established.

Reply:
@mertsincan @tugcekucukoglu @cagataycivici Yes I agree with you, I'll wait the authors of the project whether they want changing that type for accurate or not!
Thank You!